### ISSUE TITLE
NearDrop is reduce download speed of internet (somehow)

### ISSUE THREAD

**Issue by Krreedo** [grishka/NearDrop#124]
Hi! I have some strange bug - 
If I have launched neardrop app on my - mac my internet download speed is reduced drasticly, it's stuck to 6mbit max. 
For local connection all is ok (Like i am in thailand, so speed to thai servers is usual) but when i go to for exmple, London servers - it's drop to 6mbit. Right after i shutdown neardrop - all is ok. How it's happen? Can you fix this?

**Comment by heyswayam** [comment]
yes, I also noticed, it reduces the download speed. Thats what stopping me from making it as a login item orelse i would love to. Hope the dev fix this
